automol/form/_form.py

```diff
@@ -18,9 +18,10 @@
 STOICH = pp.Opt(pp.Literal("*") | ppc.integer).setParseAction(lambda x: x if x else [1])
 ATOM_COUNT = pp.Group(SYMBOL + STOICH)
 FORMULA = pp.OneOrMore(ATOM_COUNT)
+Formul = dict[str, int]
```

Comment:
Should be `Formula` (throughout). You can do this using right-click and "Rename symbol" to update all instances.

Reply:
Otherwise, the edits below look good.